core/src/main/scala/chisel3/RawModule.scala

```diff
@@ -183,6 +183,16 @@ package object internal {
     }
   }
 
+  // Sanitizes a name, e.g. from a `HasId`, by stripping all non ANSI-C characters
+  def sanitize(s: String, leadingDigitOk: Boolean = false): String = {
```

Comment:
```suggestion
  private[chisel3] def sanitize(s: String, leadingDigitOk: Boolean = false): String = {
```
package object internal is public so need to mark the method package private. Also, package object internal moved to it's own file a couple of days ago, you need to rebase.